server/utils/nunjucksSetup.ts

```diff
@@ -22,10 +22,14 @@ export default function nunjucksSetup(app: express.Express): void {
   app.use((req, res, next) => {
     const externalUser = () => req.url.startsWith('/link') || req.url.startsWith('/barcode')
     app.locals.applicationName = externalUser() ? 'Send legal mail to prisons' : 'Check Rule 39 mail'
-    res.locals.gtmContainerId = externalUser() ? config.slmContainerId : config.checkRule39ContainerId
+    app.locals.gtmContainerId = externalUser() ? config.slmContainerId : config.checkRule39ContainerId
     next()
   })
 
+  // Set the values for the phase banner and exit survey links from config
+  app.locals.phaseBannerLink = config.phaseBannerLink
+  app.locals.exitSurveyLink = config.exitSurveyLink
```

Comment:
`exitSurveyLink` is used on the 2 end stage pages (see below)
`phaseBannerLink` is already used in `layout.njk` as part of the phase banner and did not need changing:
```
{% block beforeContent %}
    {{ govukPhaseBanner({
      attributes: {
        role: "complementary"
      },
      tag: {
        text: "beta"
      },
      html: 'This is a new service – your <a class="govuk-link" rel="noreferrer noopener" target="_blank" href="'+ phaseBannerLink +'">feedback</a> will help us to improve it.'
    }) }}
  <span class="govuk-visually-hidden" id="{{ pageId }}"></span>
{% endblock %}
```